frontend/src/components/search/SearchFilterAccordion/SearchFilterSection/SearchFilterSection.tsx

```diff
@@ -95,7 +101,13 @@ const SearchFilterSection: React.FC<SearchFilterSectionProps> = ({
         // So we need hidden inputs.
         option.children?.map((child) =>
           child.isChecked ? (
-            <input key={child.id} type="hidden" name={child.value} value="on" />
+            <input
+              key={child.id}
+              type="hidden"
+              //   name={child.value}
+              name={getHiddenName(child.id)}
```

Comment:
Maybe we shouldn't do `{childrenVisible ? ( ...` but instead visually hide the children. Keep them in the DOM so that their values can be sent with the form

Reply:
Do you have pseudocode to illustrate it? This part of code was just for a weird edge case where the section is collapsed and you change some other input to cause a form submit

Reply:
I'm going to merge but I'll come back for this